src/core/server/http/router/route.ts

```diff
@@ -144,6 +144,16 @@ export interface RouteConfigOptions<Method extends RouteMethod> {
    * Additional body options {@link RouteConfigOptionsBody}.
    */
   body?: Method extends 'get' | 'options' ? undefined : RouteConfigOptionsBody;
+
+  /**
+   * Defines per-route timeouts.
+   */
+  timeout?: {
+    /**
+     * Milliseconds the socket can be idle before it's closed
+     */
+    idleSocket?: number;
```

Comment:
Would you mind renaming the `timeout` option introduced in https://github.com/elastic/kibana/pull/73103 to something like `payloadParsing`?